src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -148,6 +148,13 @@
     new DisableFlag(state -> !CONFIG_PROVIDER.getOrCreate(state).getReadBeforeWriteListOperationsEnabled(),
                     "List operation requiring read before write");
 
+    /**
+     * Guardrail disabling ALLOW FILTERING statement within a query
+     */
+    public static final DisableFlag allowFilteringEnabled =
+    new DisableFlag(state -> !CONFIG_PROVIDER.getOrCreate(state).getAllowFilteringEnabled(),
+                    "ALLOW FILTERING statement is prohibited");
```

Comment:
Hm. It seems like it'd be kind of us to have something like "Feature X is disable by guardrails", or "Only allowed to have Y of thing Z based on guardrail limit", since that implies mutability and organizational "opinion" of a setting rather than "the database just straight up doesn't do that."

More of a "all Guardrails.java" point of feedback, but could adjust at least this one to state that? WDYT?

Reply:
@josh-mckenzie I have opened [CASSANDRA-17430](https://issues.apache.org/jira/browse/CASSANDRA-17430) to consider adding a common prefix to all guardrail messages, so we don't need to mention them here.

Reply:
👍